src/Compilers/VisualBasic/Portable/Scanner/CharacterInfo.vb

```diff
@@ -488,4 +488,37 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         End Function
     End Class
 
+    Friend Module CharExts
+
+        <Runtime.CompilerServices.Extension>
+        Friend Function IsAnyOf(c As Char, c0 As Char, c1 As Char) As Boolean
```

Comment:
Considering that LNQ provides the extension method `.Any`, I don't the contextual meaning isn't that far of a leap.